deadcode.go

```diff
@@ -12,13 +12,25 @@ import (
 )
 
 var exitCode int
+var ignoreNames []string
+
+func init() {
+	flag.String("dir", ".", "directories to search, example: Foo1 Bar2")
+	flag.String("ignore", "", "sourcefile:name to ignore, example: somefile.go:aVariableFoo anotherfile.go:aConstBar")
+}
 
 func main() {
 	flag.Parse()
-	if flag.NArg() == 0 {
+
+	if flag.Lookup("ignore") != nil {
+		for _, name := range strings.Fields(flag.Lookup("ignore").Value.String()) {
+			ignoreNames = append(ignoreNames, name)
+		}
+	}
+	if flag.Lookup("dir").Value.String() == "." {
```

Comment:
My thinking was that specifying the directory where you want the job done is the main thing you want the program to do. To take an example, you wouldn't specify:

```
rm -file file1 -file file2
```

you'd specify:

```
rm file1 file2
```

Do that make it clearer?